python/ray/data/_internal/util.py

```diff
@@ -136,7 +136,7 @@ def _autodetect_parallelism(
     return parallelism, min_safe_parallelism
 
 
-def _estimate_avail_cpus(cur_pg: Optional["PlacementGroup"]) -> int:
+def _estimate_avail_cpus(cur_pg: Optional["PlacementGroup"] = None) -> int:
```

Comment:
Could we retain this argument? It is more likely you can forget to pass in this otherwise.

Reply:
Let me change the call to `_estimate_available_parallelism()`, that's really what I was looking for, and I'll revert this change!